src/polyfem/solver/forms/adjoint_forms/TransientForm.cpp

```diff
@@ -180,7 +180,7 @@ namespace polyfem::solver
 	void TransientForm::solution_changed(const Eigen::VectorXd &new_x)
 	{
 		AdjointForm::solution_changed(new_x);
-		obj_->solution_changed(new_x);
+		// obj_->solution_changed(new_x);
```

Comment:
Like mentioned above, can we remove "solution_changed_step" and use "solution_changed" instead?